src/pybind/mgr/dashboard/run-promtool-unittests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+: ${CEPH_ROOT:=$SCRIPTPATH/../../../../}
+
+docker run --rm \
+    -v "$CEPH_ROOT":/ceph \
+    --name=promtool \
+    --network=host \
+    dnanexus/promtool:2.9.2 \
+    test rules /ceph/monitoring/prometheus/alerts/test_alerts.yml
```

Comment:
I'd rather put this into the dashboard tox file. We were already using tox for checking Grafana dashboards (`check` toxenv).